lang/dart.md

```diff
@@ -0,0 +1,204 @@
+---
+lang: Dart
+syntax: Dart
+title: Dart is Smart Language!
+author: chandan0000
+---
+
+```dart
+void main(){
+    print("hello world");
+}
+```
+## What is Dart?
+Dart is a new general and open source programming language with a vibrant community developed by Google Inc. and its official website is https://dart.dev. It was first announced as a public preview on October 10, 2011; it has now reached version 1.10. World class language designers and developers are involved in this project, namely, Lars Bak and Kasper Lund (known for their V8 JavaScript engine embedded in the Chrome browser, which revolutionized performance in the JavaScript world), and Gilad Bracha (a language theorist known for the development of the Strongtalk and Newspeak languages and for the Java specification). Judging by the huge amount of resources and the number of teams working on it, it is clear that Google is very serious about making Dart a success.
```

Comment:
"Dart is a new general and open source programming lang" is not a good headstart!

Pl provide the reference, from where did you get these words/sentences.

References:
- [Wiki](https://en.wikipedia.org/wiki/Dart_(programming_language))
- [Google Codelabs](https://codelabs.developers.google.com/codelabs/from-java-to-dart#0)

Take the reference from here, will be more beneficial!